pippo-core/src/main/java/ro/pippo/core/Application.java

```diff
@@ -248,7 +249,8 @@ public final RequestResponseFactory getRequestResponseFactory() {
      * @return
      */
     protected RequestResponseFactory createRequestResponseFactory() {
-        return new RequestResponseFactory(this);
+//        return new RequestResponseFactory(this);
+        return new GZipRequestResponseFactory(this);
```

Comment:
I don't know if it's a good idea or not to have `GZipRequestResponseFactory` (gzip encoding) as default (instead of simple `RequestResponseFactory`). What do you say?